Quaver.Shared/Skinning/SkinKeys.cs

```diff
@@ -543,8 +547,16 @@ private void LoadLaneSpecificElements()
                     ColumnColors[i] = ConfigHelper.ReadColor(ColumnColors[i], Store.Config[ModeHelper.ToShortHand(Mode).ToUpper()][$"ColumnColor{i + 1}"]);
 
                 // HitObjects
-                LoadHitObjects(NoteHitObjects, $"note-hitobject-{i + 1}", i);
-                LoadHitObjects(NoteHoldHitObjects, $"note-holdhitobject-{i + 1}", i);
+                if (UseSameHitObjects == true)
+                {
+                    LoadHitObjects(NoteHitObjects, $"note-hitobject-{i + 1}", i);
+                }
+                else
```

Comment:
Also, if you're using the same textures, there's no need to load them in twice. You can use the same objects.

Reply:
Not sure how to do that.

Reply:
`NoteHoldHitObjects = NoteHitObjects`